docs/hub/enterprise-sso.md

```diff
@@ -6,6 +6,8 @@ This feature is part of the <a href="https://huggingface.co/enterprise">Enterpri
 
 Single sign-on (SSO) allows organizations to securely manage user authentication through their own identity provider (IdP). Both SAML 2.0 and OpenID Connect (OIDC) protocols are supported.
 
+Please note that SSO on Hugging Face is designed for managing access to organization-specific resources (like private models, datasets, and Spaces) and does not replace the core authentication for the Hugging Face platform itself.
```

Comment:
I think with User provisioning, at least SCIM (maybe JIT) it can replace authentication? (cc @Kakulukian)